Add term entries for service messages (#317)
raiden-network/spec: terminology.rst

   expiration
        Specific block after which the lock in the :term:`locked transfer` expires

   MonitoringRequest

>         A message that is broadcasted from a client to :term:`Monitoring Service` in order to provide the MS with the necessary information to watch the channel.

Reviewer: ```suggestion
        A message that is broadcasted from a client to :term:`Monitoring Services<Monitoring Service>` in order to request channel watching from the MS. The message contains all the necessary information for the MS to carry out a channel update for the user.
```

   PFSCapacityUpdate

>         A message that is broadcasted from the client to the :term:`Pathfinding Service` in order to update the channel capacity.

Reviewer: ```suggestion
        A message that is broadcasted from the client to the :term:`Pathfinding Services<Pathfinding Service>` in order to update the channel capacity.
```

   PFSFeeUpdate

>         A message that is broadcasted from the client to the :term:`Pathfinding Service` in order to update the mediation fees.

Reviewer: ```suggestion
        A message that is broadcasted from the client to the :term:`Pathfinding Services<Pathfinding Service>` in order to update the mediation fees demanded by the client.
```

Use wording from docstring.
